packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,155 @@
+
+```html
+<nimble-accordion appearance="block">
+    <nimble-accordion-item>
+        <div slot="heading">Accordion one</div>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion one content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion two</div>
+        <nimble-button>Text Button</nimble-button>
+        Accordion two content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion three</div>
+        <nimble-checkbox></nimble-checkbox>
+        <nimble-accordion>
+            <nimble-accordion-item>
+                <div slot="heading">Nested Accordion One</div>
+                Nested Accordion One Content
+            </nimble-accordion-item>
+            <nimble-accordion-item>
+                <div slot="heading">Nested Accordion Two</div>
+                Nested Accordion Two Content
+            </nimble-accordion-item>
+        </nimble-accordion>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion four</div>
+        <nimble-button>Text Button</nimble-button>
+        Accordion four content
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+    -   FAST's attribute `expand-mode` will be omitted from the FAST base class due to issues discussed below.
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. These designs are similar to those of the nimble-button and its `appearance` types, but the accordion does not use the `.control` css class, it uses `.region` and `.heading`. Because of this, differences in border functionality, and the possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion and accordion-item will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. `:host([appearance='outline'])` will have styling for the outline button).
+
+Documentation will be added to advise against using multiple Nimble Accordions with different appearances next to each other.
+
+When using nested accordions, the parent accordion will have the selected `appearance` type, and all children accordions will be forced to have the `ghost` appearance type.
```

Comment:
I'm concerned a little about how this is described. It may be updated based on implementation but generally we want to avoid a situation where we are manipulating attributes that are owned by the user.

So we wouldn't for example want a parent accordion setting the actual appearance attribute on a child accordion. You can end up with a conflict in state where the user expects the attribute to be set one way and we are fighting the value they set and overwriting it with another.

Instead we would want something out of band. Not communicating with public attributes. Alternatively we say that complexity isn't worth it and we just document that child accordions should be configured as ghost by the user.

Spec-wise I think you should say if it is enforced via docs or by the components is an open question.

Reply:
There were several questions brought up at stand-up that need to be addressed with Brandon because they can have a large impact on implementation direction:

- Do we need the block appearance mode behavior where nested accordion headers extend all the way to the edge while nested accordion items are indented. (That requirement may require creation of a nested-accordion element to accomplish, may require more research)

Design questions:
- Should the hover borders appear around the full accordion items or just the header. Concerns that it is not clearly indicating what part of the control you can interact with. Concerns that it creates visual noise particularly with accordions with many / long accordion-items.
- Do we have use cases for the different appearance modes? Is there a discussion of just having block (for the outermost accordion) and ghost (for nested accordions)? What's the use case for outline accordions?
- Should outline accordions only outline the header (as block accordions only show the block background for the header)?

Moving back to draft per new contribution policy as clarifications about nested sub accordion requirements could result in more significant changes (creation of additional elements and maybe some prototyping).

Reply:
These questions were discussed with Brandon and overall, it seems that the Accordion will be put to the side for now and revisited at a later date due to other priorities, the need for an IxD, and the need for more specific accordion UX designs based on the questions presented.

Some possible paths to take were discussed, including:
- It's currently unclear how appearances other than the ghost border will function inside of nested accordions, as UX designs for these cases aren't fully created yet.
- The hover border should only appear around the accordion header (if we decide to keep a hover animation at all, as many other examples of accordions do not have a hover animation).
- Thoughts on keeping different appearance modes are still unclear, but it's been discussed that designers would benefit from having multiple appearance options available when using accordions.
- I don't believe the question regarding outlines around only the header was discussed, but if we're changing the hover animation to only show up around the header, then I would presume that the outline border would follow suit as a green hover border taking up only the header portion of the outline in the expanded mode (instead of the entire outline border) would look odd.